tests/inspector/ckeditorinspector.js

```diff
@@ -41,27 +45,48 @@ describe( 'CKEditorInspector', () => {
 	describe( '#attach()', () => {
 		it( 'adds inspector to DOM', () => {
 			expect( document.querySelector( '.ck-inspector-wrapper' ) ).to.be.null;
+			expect( CKEditorInspector._wrapper ).to.be.null;
 
-			CKEditorInspector.attach( 'foo', editor );
+			CKEditorInspector.attach( { foo: editor } );
 
-			const wrapper = document.querySelector( '.ck-inspector-wrapper' );
+			expect( CKEditorInspector._wrapper ).to.be.instanceOf( HTMLElement );
+
+			const wrapper = CKEditorInspector._wrapper;
 
 			expect( wrapper.tagName.toLowerCase() ).to.equal( 'div' );
 			expect( wrapper.parentNode ).to.equal( document.body );
 			expect( wrapper.childNodes.length ).to.equal( 2 );
 			expect( wrapper.firstChild.classList.contains( 'ck-inspector' ) ).to.be.true;
 		} );
 
-		it( 'attaches editors under generated names', () => {
+		it( 'adds inspector to DOM only once when attaching to the first editor', () => {
 			return TestEditor.create( element )
```

Comment:
I'd add `document.createElement( 'element2' )`, this will fail with the classic editor.